### PR TITLE
Add/update apps

### PR DESCRIPTION
**Pull request by hassankhan**
- Android Studio
- Colima
- Crossover
- Dolt CLI
- Facebook Flipper
- Homebridge
- iTerm2
- JetBrains Fleet
- Lima
- Node
- pnpm
- Proxyman
- Pulumi CLI
- Redis Insight
- usql CLI
- Vue CLI

### CODE REVIEW

File: programs/crossover.json
Comment (hassankhan) [inline programs/crossover.json:1]: These files may actually be created by Wine (used under-the-hood by Crossover)

File: programs/redisinsight.json [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+            "movable": true,
+            "help": "Export the following environment variables:\n\n```bash\nexport RI_APP_FOLDER_ABSOLUTE_PATH=\"$XDG_DATA_HOME\"/redisinsight-app\n```\n"
+        },
         {
             "path": "$HOME/.redisinsight-v2",
             "movable": true,
             "help": "Export the following environment variables:\n\n```bash\nexport APP_FOLDER_ABSOLUTE_PATH=\"$XDG_DATA_HOME\"/redisinsight-v2\n```\n"
         },
         {
-            "path": "$HOME/..redisinsight-preview",
```
Comment (hassankhan) [inline programs/redisinsight.json:9]: This seemed like a typo

**Comment by hassankhan** [comment]
@b3nj5m1n I've fixed the broken JSON, could you re-run the workflow when you get a moment please? Thanks in advance 😀

**Comment by b3nj5m1n** [comment]
Thank you very much for all of these!

**Pull request merged**